src/amo/api/reviews.js

```diff
@@ -186,37 +186,36 @@ export function getReviews({
   });
 }
 
-export type GetLatestReviewParams = {|
+export type GetLatestUserReviewParams = {|
   addon: number,
```

Comment:
I know this is old code, but it is a bit counter-intuitive. Should we be using `addonId` and `userId` here instead of `addon` and `user`?

Reply:
Since it's an API function that maps directly to [this `addon` parameter](https://addons-server.readthedocs.io/en/latest/topics/api/ratings.html#list-ratings), the value can be either an `addonId` or an `addonSlug`.

Reply:
Whoops, this should have been `addon: number | string`

Reply:
That makes more sense.